### ISSUE TITLE
MobileNetV2 for object_detection

### ISSUE THREAD

**Issue by GustavZ** [tensorflow/models#3804]
### System information
- **What is the top-level directory of the model you are using**: object_detection

Are there any plans to include mobilenetV2 into object_detection in the nearer future?
Also it would be interesting to use it for semantic segmentation, will this be included?

**Comment by lcycoding** [comment]
The config file for training SSD in mobilenetv2 backbone is available in research/object_detection/samples/config for a while.

Hope it helps.

**Comment by Johnson-yue** [comment]
@lcycoding hi, I also see the config file for fine-tuning mobilenetv2 model. But there is not ssd_mobilev2_coco.ckpt for _Fine_tuning_. I don`t  know whether can be training from scracth by using object detection Model

**Comment by lcycoding** [comment]
@Johnson-yue I think tensorflow team will release that in a near future?
But I do run train-from-scratch process using certain config file, it'll converge in some way. lol

**Comment by GustavZ** [comment]
do you guys have a clue how mobilenetv2 can be used for mask segmentation as written in the paper?

**Comment by lcycoding** [comment]
@GustavZ  Sorry, I haven't tried yet.
But certain setting does exist in the feature extractor code. You could give it a try.
